addons/account/__manifest__.py

```diff
@@ -55,7 +55,7 @@
         'data/payment_receipt_data.xml',
         'wizard/setup_wizards_view.xml',
         'views/account_onboarding_templates.xml',
-        'data/service_cron_reverse.xml',
+        'data/service_cron.xml',
```

Comment:
that file doesn't appear anymore as renamed in github, is it a bug?

Reply:
Eh... I just did a `git commit --amend` so it should't have happened I think ?

Reply:
ok if you didn't change anything, it's only github that mess it